redis/Cargo.toml

```diff
@@ -41,7 +43,6 @@ tokio-util = { version = "0.7", optional = true }
 tokio = { version = "1", features = ["rt", "net", "time"], optional = true }
 
 # Only needed for the connection manager
-arc-swap = { version = "1.1.0", optional = true }
```

Comment:
this should remain optional, just used in aio instead of connection-manager.

Reply:
as you will see in push_manager.rs it's used in PushManager.
btw i didn't use mutex because then the code will get messy with async runtimes.